test/test_helpers.h

```diff
@@ -85,6 +142,116 @@ inline void cube_mesh_cw(const Vec3r<T> &center, Vec3rList<T> &vertices, Vec3iLi
   indices.emplace_back(Vec3ui{0, 1, 3});
 }
 
+template<typename T>
+inline void two_plane_mesh_90_deg(const Vec3r<T> &center, Vec3rList<T> &vertices, Vec3iList &indices) {
```

Comment:
please document what this function creates

Reply:
done